test/verify/check-system-services

```diff
@@ -299,6 +299,8 @@ Unit=test.service
         self.check_service_details(["Disabled"], ["Start", "Disallow running (mask)"], False)
         self.toggle_onoff()
         self.check_service_details(["Failed to start", "Automatically starts"], ["Start", "Clear 'Failed to start'", "Disallow running (mask)"], True)
+        # ignore logs panel
+        b.assert_pixels("#service-details", "details-test-fail", ignore=[".contains-list"])
```

Comment:
That would be nice, but the log table is really just a `<div class="pf-c-card__body contains-list">`. You are suggesting to ignore the whole card -- I wanted to keep the card frame and title in the screenshot, otherwise I could just snapshot only the first card.

Reply:
Ignore my previous comment, I mis-read. Indeed that makes sense, and we already do that in check-networkmanager-basic. Done.